Source/WebCore/dom/TrustedType.cpp

```diff
@@ -173,6 +187,39 @@ ExceptionOr<String> trustedTypeCompliantString(TrustedType expectedType, ScriptE
     return stringValue;
 }
 
+String getTrustedTypeForAttribute(const String& tagName, const String& attributeParameter, const String& elementNamespace, const String& attributeNamespace, String sink)
```

Comment:
Change this to return `TrustedType` enum rather than a string (you just convert it back anyway)

Reply:
As discussed, we leave it as String for now and do the clean up at a later stage.